theatres/aliabadregion-2024/TCO30_Austin_JanitorialDuty.MCN_Aliabad/initPlayerLocal.sqf

```diff
@@ -0,0 +1,26 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+target1 setName (shuffledTargets select 0);
+target2 setName (shuffledTargets select 1);
+target3 setName (shuffledTargets select 2);
```

Comment:
Even though this should be fine since initServer.sqf is run first, I would turn this into a CBA event handler, and call `CBA_fnc_globalEventJIP` in `initServer.sqf`.

```suggestion
["mission_setName", {
  params ["_unit", "_name"];
  _unit setName _name;
  _unit setVariable ["ACE_name", _name#0];
  _unit setVariable ["ACE_nameRaw", _name#0];
}] call CBA_fnc_addEventHandler;
```